frontend/src/features/landing_page/components/GeoMap.tsx

```diff
@@ -47,4 +50,29 @@ export const GeoMap = () => {
       {markers}
     </MapContainer>
   );
+
+  function handleClick() {
+    setIsVisible(true);
+  }
+
+  let content;
+
+  if (isVisible) {
+    content = map;
```

Comment:
```suggestion
  let content;

  if (isVisible) {
    content = map;
```
Instead of doing this we could add an early return.
```ts

if (!isVisible) {
  // the placeholder from below. 
  return placeholder;
}

return map;

Reply:
You're right, I totally didn't think of that.